demisto_sdk/commands/common/tools.py

```diff
@@ -3530,47 +3530,52 @@ def get_display_name(file_path, file_data={}) -> str:
     :rtype: ``str``
     :return The display name
     """
-    if not file_data:
-        file_extension = os.path.splitext(file_path)[1]
-        if file_extension in [".yml", ".json"]:
-            file_data = get_file(file_path)
-
-    if "display" in file_data:
-        name = file_data.get("display", None)
-    elif "layout" in file_data and isinstance(file_data["layout"], dict):
-        name = file_data["layout"].get("id")
-    elif "name" in file_data:
-        name = file_data.get("name", None)
-    elif "TypeName" in file_data:
-        name = file_data.get("TypeName", None)
-    elif "brandName" in file_data:
-        name = file_data.get("brandName", None)
-    elif "id" in file_data:
-        name = file_data.get("id", None)
-    elif "trigger_name" in file_data:
-        name = file_data.get("trigger_name")
-    elif "rule_name" in file_data:
-        name = file_data.get("rule_name")
+    _file_path = Path(file_path)
+    _file_data: dict = file_data or {}
```

Comment:
I've added a separate `_file_data` parameter since mypy complained for the section below that `file_data` can be `None` (due to the typing of the function - `file_data: dict | None = None`)

Reply:
then just do `file_path = Path(file_path)` - no need for new variable if you define it to be either `str | Path`
also, you can do for `_file_data`
```
if not file_data:
    file_data = {}
```

also an option instead of creating new var